docs/pages/docs/guides/workspaces.mdx

```diff
@@ -0,0 +1,251 @@
+---
+title: Workspaces
+description: A guide on workspaces to onboard onto using Turborepo.
+---
+
+import {Tabs, Tab} from '../../../components/Tabs'
+
+# Workspaces
+
+Turborepo makes use of workspaces to organize your apps and packages. But how do they work? This guide will walk you through the basics of workspaces.
+
+## What are workspaces?
+
+Workspaces are a feature implemented by package managers to assist with working on multiple apps and packages in the same codebase.
+Turborepo is compatible with 3 different package managers. Each with its own implementation of workspaces, links to their respective documentation are listed below. Please note that there are slight nuances between implementations and functionality.
+
+- [npm workspaces](https://docs.npmjs.com/cli/v7/using-npm/workspaces)
+- [yarn workspaces](https://classic.yarnpkg.com/lang/en/docs/workspaces/)
+- [pnpm workspaces](https://pnpm.io/workspaces)
+
+Workspaces allow you to create folders, each called a workspace, each with its own package.json, and you can install packages into each workspace.
+A workspace can also depend on other workspaces.
+A workspace is generally either an app, an application that can be executed such as an api, or a package, an application that can be used as a dependency by other workspaces.
+
+## Working with workspaces
+
+### Setup workspaces
+
+When using workspaces you'll first have to define the location of your workspaces.
+That is the folders that your package manager should look in for workspaces.
+
+A common pattern is to have an `apps` folder and a `packages` folder.
+The `apps` folder will contain workspaces that house launchable apps, such as a next.js app.
+The `packages` folder will contain workspaces that house packages that are used by either an app or another package.
+
+The configuration for this is different for `pnpm` than it is for `npm` and `yarn`.
+
+<Tabs items={['npm & yarn', 'pnpm']} storageKey="workspace-config-for-different-package-managers">
```

Comment:
mimic the other tabs usage and duplicate content for npm and yarn so that we can use the same storage key as getting-started.mdx